apps/ledger-live-mobile/src/families/mina/SendRowMemo.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback } from "react";
+import { View, StyleSheet } from "react-native";
+import { Trans } from "react-i18next";
+import { useNavigation, useRoute, useTheme } from "@react-navigation/native";
+import type { Account } from "@ledgerhq/types-live";
+import type { Transaction as MinaTransaction } from "@ledgerhq/live-common/families/mina/types";
+import LText from "~/components/LText";
+import { ScreenName } from "~/const";
+import SummaryRow from "~/screens/SendFunds/SummaryRow";
+import { BaseComposite, StackNavigatorProps } from "~/components/RootNavigator/types/helpers";
+import { SendFundsNavigatorStackParamList } from "~/components/RootNavigator/types/SendFundsNavigator";
+import { SignTransactionNavigatorParamList } from "~/components/RootNavigator/types/SignTransactionNavigator";
+import { SwapNavigatorParamList } from "~/components/RootNavigator/types/SwapNavigator";
+
+type Navigation = BaseComposite<
+  | StackNavigatorProps<SendFundsNavigatorStackParamList, ScreenName.SendSummary>
+  | StackNavigatorProps<SignTransactionNavigatorParamList, ScreenName.SignTransactionSummary>
+  | StackNavigatorProps<SwapNavigatorParamList, ScreenName.SwapSelectFees>
+>;
+
+type Props = {
+  account: Account;
+  transaction: MinaTransaction;
+} & Navigation;
+export default function MinaMemoRow({ account, transaction }: Props) {
+  const { colors } = useTheme();
+  const navigation = useNavigation<Navigation["navigation"]>();
+  const route = useRoute<Navigation["route"]>();
+  const editMemo = useCallback(() => {
+    navigation.navigate(ScreenName.MinaEditMemo, {
+      ...route.params,
+      accountId: account.id,
+      parentId: undefined,
+      account,
+      transaction,
+    });
+  }, [navigation, route.params, account, transaction]);
+  const memo = transaction.memo;
+  return (
+    <View>
+      {!memo ? (
+        <SummaryRow title={<Trans i18nKey="send.summary.memo.title" />} onPress={editMemo}>
+          <LText
+            style={[
+              styles.link,
+              {
+                textDecorationColor: colors.live,
+              },
+            ]}
+            color="live"
+            onPress={editMemo}
+          >
+            <Trans i18nKey="common.edit" />
+          </LText>
+        </SummaryRow>
+      ) : (
+        <SummaryRow title={<Trans i18nKey="common.edit" />} onPress={editMemo}>
+          <LText semiBold style={styles.tagText} onPress={editMemo}>
+            {String(memo)}
```

Comment:
If we do String(undefined) it shows undefined, maybe we need to deal with edge cases ?
(or atleast deal with it in logic part)

Reply:
memo should always be a string if it reaches there because of this check https://github.com/LedgerHQ/ledger-live/pull/9230/files/e907a975b320513dcdca988391597fcb9fbd7d55#diff-9633049423ed53b6f54cc3f749e88283804d1644c38b4c8ecf2005722ac86412R41

Reply:
I cannot see your comment here @semeano, I think due to rebase. Can you please repost? 🙏🏻

Reply:
The link you gave does not go anywhere. It can't find the commit.

Reply:
Ahh, I see... you send an image but it blends so well with the background it feels like a legit error message.

Sorry about that, I meant check on line 41 `!memo` with the ternary operator.
<img width="579" alt="Screenshot 2025-03-26 at 21 57 45" src="https://github.com/user-attachments/assets/290cc186-f22b-44e0-be67-32c6e3cdffdb" />